src/components/ItemCell/index.tsx

```diff
@@ -234,11 +231,18 @@ export default memo(function ItemCell({
       {equippedByOtto && (
         <StyledEquipped>
           <Help noicon message={equippedByOtto.name}>
-            <Link href={`/my-ottos/${equippedByOtto.id}`} passHref>
+            {!currentOtto && (
+              <Link href={`/my-ottos/${equippedByOtto.id}`} passHref>
+                <StyledOttoLink onClick={e => e.stopPropagation()}>
+                  <Image src={equippedByOtto.image} layout="responsive" width={24} height={24} />
+                </StyledOttoLink>
+              </Link>
+            )}
+            {currentOtto && (
               <StyledOttoLink onClick={e => e.stopPropagation()}>
```

Comment:
```suggestion
              <StyledOttoLink as="span">
```

I this case, it's not a link anymore.

Reply:
Done